x/rewards/genesis.go

```diff
@@ -19,6 +19,10 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	if len(genState.RefillRewardsTS.TimeEntries) == 0 {
 		k.RefillRewardsPools(ctx, nil, nil)
 	}
+	for _, iprpcReward := range genState.IprpcRewards {
```

Comment:
you dont need to do this if you do k.SetIprpcData(ctx, genState.MinIprpcCost, genState.IprpcSubscriptions)

Reply:
this is not true. The IPRPC data is not related to the IprpcReward objects